docs/man_pages/project/testing/appstore-upload.md

```diff
@@ -0,0 +1,52 @@
+appstore upload
+==========
+
+Usage | Synopsis
+---|---
+General | `$ tns appstore upload [<Apple ID> [<Password> [<Mobile Provisioning Profile Identifier> [<Code Sign Identity>]]]]]`
+General | `$ tns appstore upload [<Apple ID> [<Password>]] --ipa <Ipa File Path>`
```

Comment:
Instead of two Generals, go with:

Build and upload package | blah blah
Upload package | blah blah